use-timescale/backup-restore/point-in-time-recovery.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Point-in-time recovery
+excerpt: Learn how to restore your service to a previous point in time
+products: [cloud]
+keywords: [restore, recovery, pitr]
+tags: [recovery, restore, pitr]
+---
+
+# Point-in-time recovery
+
+Timescale allows you to perform a point-in-time recovery of your service to any 
+point in the last 3 days. Point-in-time recovery allows you to recover your 
+database from a destructive or unwanted action or change. Initiating a 
```

Comment:
Should the first two sentences be switched?  First describe PITR, then how Timescale's PITR works